app/controllers/api/payment/braintree_controller.rb

```diff
@@ -14,6 +15,11 @@ def webhook
     end
   end
 
+  def one_click
+    BraintreeServices::OneClickService.new(params).run
```

Comment:
I'm starting to get lost with the number of services we have - it'd be nice to get together and draw a chart / dependency tree of services, classes, modules...